freqtrade/rpc/rpc.py

```diff
@@ -521,6 +523,17 @@ def _rpc_trade_statistics(
 
         profit_factor = winning_profit / abs(losing_profit) if losing_profit else float('inf')
 
+        mean_winning_profit = (winning_profit / winning_trades) if winning_trades > 0 else 0
+        mean_losing_profit = (abs(losing_profit) / losing_trades) if losing_trades > 0 else 0
+
+        winrate = (winning_trades / closed_trade_count) * 100 if closed_trade_count > 0 else 0
+        loserate = (100 - winrate)
```

Comment:
These 2 variables should be as ratio (0-1), not as % (0-100)

winrate is multiplied by 100 - and subsequently both values are `/100` seveal times for subsequent calculations.
For an API consumer, it's also simpler to use it if it's a ratio - showing "52%" is really a formatting question, not how the value should bein the API - and in python, can be Accomplished quite easily with `:.2%}` instead of `:.2f}%`.

Reply:
Done!